src/js/components/Breadcrumb.js

```diff
@@ -46,6 +72,12 @@ class Breadcrumb extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
+    let {buildCrumbCustomParams} = this.props;
+
+    if (!DeepEqual(nextProps.buildCrumbCustomParams, buildCrumbCustomParams)) {
```

Comment:
No it just defaulted to this order :) If it works in the other order, lets do it!